lamindb_setup/core/_clone.py

```diff
@@ -91,3 +104,115 @@ def connect_local_sqlite(instance: str) -> None:
     isettings = load_instance_settings(settings_file)
     isettings._persist(write_to_disk=False)
     isettings._load_db()
+
+
+def connect_remote_sqlite(
+    instance: str, *, copy_suffix: str | None = None, overwrite: bool = False
+) -> None:
+    """Load a remote SQLite instance of which a remote hub Postgres instance exists.
+
+    This function is the main building block for loading remote clones.
+
+    Args:
+        instance: Instance slug in the form `account/name` (e.g., `laminlabs/privatedata-local`).
+        copy_suffix: Optional suffix of the local clone.
+        overwrite: Whether to overwrite existing local database.
+    """
+    import lamindb_setup as ln_setup
+
+    # Step 1: Create the settings file
+    # We need to connect to the real instance to get the settings
+    ln_setup.connect(instance)
```

Comment:
do this instead https://github.com/laminlabs/lamindb-setup/blob/881a219a1a99d21bb22efa17f9198067168212c7/lamindb_setup/_connect_instance.py#L100